### ISSUE TITLE
FargateCluster defaults require too many VCPUs for a new AWS account

### ISSUE THREAD

**Issue by adair-kovac** [dask/dask-cloudprovider#299]
I created a new AWS account to experiment with dask, but I can't create a cluster with any workers without specifying CPU settings:

```
from dask_cloudprovider.aws import FargateCluster
cluster = FargateCluster(n_workers=1)
```
Output:

/opt/anaconda3/lib/python3.8/contextlib.py:120: UserWarning: Creating your cluster is taking a surprisingly long time. This is likely due to pending resources on AWS. Hang tight! 
  next(self.gen)
Task exception was never retrieved
future: <Task finished name='Task-1259' coro=<_wrap_awaitable() done, defined at /opt/anaconda3/lib/python3.8/asyncio/tasks.py:688> exception=RuntimeError({'tasks': [], 'failures': [{'reason': **'The requested CPU configuration is above your limit'**}], 'ResponseMetadata': {'RequestId': '3e739dbb-4c9c-4d4b-b37d-a02ef76897c9', 'HTTPStatusCode': 200, 'HTTPHeaders': {'x-amzn-requestid': '3e739dbb-4c9c-4d4b-b37d-a02ef76897c9', 'content-type': 'application/x-amz-json-1.1', 'content-length': '90', 'date': 'Thu, 01 Jul 2021 18:36:07 GMT'}, 'RetryAttempts': 0}})>

I'm not sure *why* it's happening given the defaults should work out to 5 vCPUs and my account limit is 5 vCPUs, and I did check to make sure I didn't have anything else running. Maybe there's some overhead, maybe it's user error.

The fix seems easy enough:

```
from dask_cloudprovider.aws import FargateCluster
cluster = FargateCluster(n_workers=1, worker_cpu=256, worker_mem=512)
```

but as a brand-new user, this meant I had to spend the rest of my day learning about AWS instead of continuing with the tutorial. 

Searching for that error just brings up this [thread](https://forums.aws.amazon.com/thread.jspa?threadID=335825) where a ton of other users collectively fail to figure out which service quota limit it is they have to ask for an increase in, so it would also be nice if the documentation pointed new users in the right direction though I get it if that's outside the scope of the dask project.

**Comment by jacobtomlinson** [comment]
Thanks for raising this @adair-kovac, sorry you had trouble here.

Our defaults should definitely be below the new account maximum, I can imagine others would also have this problem. We should drop the defaults.

Adding a section to the [troubleshooting docs](https://github.com/dask/dask-cloudprovider/blob/main/doc/source/troubleshooting.rst) may help make things more googleable here.

As you've recently experienced this and know which documentation was most helpful to you do you have any interest in raising a PR to add that?

**Comment by adair-kovac** [comment]
@jacobtomlinson Of course! Can you confirm that the correct service quota would be "Running On-Demand Standard (A, C, D, H, I, M, R, T, Z) instances"? I had that raised and just verified that I can now create a larger cluster, but I also requested some other limits to be raised so I want to be sure this is the correct one.

**Comment by jacobtomlinson** [comment]
@adair-kovac honestly I'm not sure
